packages/SwingSet/src/vats/vat-timerWrapper.js

```diff
@@ -26,8 +28,9 @@ export function buildRootObject(vatPowers) {
       },
       makeRepeater(delaySecs, interval) {
         Nat(delaySecs);
+        Nat(interval);
         assert(
-          Nat(interval) > 0,
+          interval > 0,
```

Comment:
Can this second assert ever get triggered? I think the new `Nat(interval)` call added above will fire first. It'd be nice if there were a way to use `Nat(interval)` as an assertion but still print some details, but maybe it's sufficient for someone to look at the stack trace and source code to figure out that the `Nat()` which failed was for the second argument, rather than the first.

Reply:
Nat includes 0. So the second assertion is triggered if it equals 0.